### PR TITLE
CV styled

### PR DESCRIPTION
**Pull request by b-rage**

### CODE REVIEW

File: staff/gianluca-bragaglia/cv-styled/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        <address>
+            151 East 2º street <br> 
+            New York 12345 <br>
+            <a href="mailto:info@dor.com">info@dor.com</a>
+        </address>
+        <br>
+        <hr>
+    </header>
+    <br>
+    <div class="container">
```
Comment (vior1992) [inline staff/gianluca-bragaglia/cv-styled/index.html]: Maybe better use "MAIN" tan "DIV" for this section?

Comment (b-rage) [reply staff/gianluca-bragaglia/cv-styled/index.html]: I have 6 div container, I think is not good to put 6 main container

Comment (vior1992) [reply staff/gianluca-bragaglia/cv-styled/index.html]: I mean, all this part inside a main, my bad

Comment (b-rage) [reply staff/gianluca-bragaglia/cv-styled/index.html]: fixed

**Pull request merged**